### PR TITLE
Adds log to display filename

### PR DESCRIPTION
**Pull request by srbcheema1**
This PR is for the issue #115
I have added a message to display the filename of output file

### CODE REVIEW

File: src/validator_main.cpp [hunk 1]
```diff
@@ -189,6 +189,7 @@ int main(int argc, char** argv)
         std::string report_result = "According to the VCF specification, the input file is " + std::string(is_valid ? "" : "not ") + "valid";
         BOOST_LOG_TRIVIAL(info) << report_result;
         for (auto & output : outputs) {
+            BOOST_LOG_TRIVIAL(info) << "Output File : " << output->get_filename();
```
Comment (jmmut) [inline src/validator_main.cpp]: the message is a bit more clear as `"Report written to: "`.

**Comment by srbcheema1** [comment]
@jmmut I have made the requested changes.
Please have a look at it.

**Comment by srbcheema1** [comment]
@jmmut I think there may be possibility that some error or exception is generated in try block before writing it to file. so thats why i was preffering not to print it in line 143. As it would be pointless to print it there if no file is written. tell me if I got something wrong.

**Comment by jmmut** [comment]
oh, my fault, the line `std::string report_result = "...` needs to be before the loop

**Comment by jmmut** [comment]
regarding the log placement, we are discussing it, we'll tell you later

**Comment by jmmut** [comment]
ok, after thinking about this, just move the line `std::string report_result = "...` line before the loop to make it compile and we'll merge this. In the future, if we have another report that doesn't write to a file we'll change the name of the method to something more generic, because it will be still useful.

**Comment by srbcheema1** [comment]
@jmmut , should I undo the changes I did in last commit regarding these two lines(moving both the lines together). Or should I move only-begotten one line above the loop.

**Comment by jmmut** [comment]
only one line, we still want the last line to be the final yes/no statement.

**Pull request merged**